python/pyspark/sql/streaming.py

```diff
@@ -51,22 +51,22 @@ def isActive(self):
         return self._jcq.isActive()
 
     @since(2.0)
-    def awaitTermination(self, timeoutMs=None):
+    def awaitTermination(self, timeout=None):
         """Waits for the termination of `this` query, either by :func:`query.stop()` or by an
         exception. If the query has terminated with an exception, then the exception will be thrown.
-        If `timeoutMs` is set, it returns whether the query has terminated or not within the
-        `timeoutMs` milliseconds.
+        If `timeout` is set, it returns whether the query has terminated or not within the
```

Comment:
It might be good to mention what is returned if we time out (as we do in `awaitTerminationOrTimeout` on the `StreamingContext`).